### PR TITLE
Several updates to our security docs:

### PR DESCRIPTION
**Pull request by alex**
- Document what we consider a vulnerability
- Update my prefered PGP key identifier
- Clarify that we only provide security support for the most recent release (this has empirically been true for all of our releases)

### CODE REVIEW

File: docs/security.rst [hunk 1]
```diff
@@ -5,6 +5,39 @@ We take the security of ``cryptography`` seriously. The following are a set of
...
+  confidentiality or integrity with the public API (e.g. if the API let a user
+  reuse nonces).
+* If, under any circumstances, we used a CSPRNG which wasn't fork-safe.
+* If ``cryptography`` used an API in an underlying C library and failed to
+  handle error conditions safely.
+
+Examples of things we wouldn't consider security issues:
+
+* Offering ECB mode for symmetric encryption. Though ECB is critically weak, it
+  is documented as being weak in our documentation.
```
Comment (dstufft) [inline docs/security.rst]: Should this explicitly mention offering ECB *in the hazmat layer*? I believe we would treat it as a security issue if the recipes layer allowed you to use ECB somehow yea?

Comment (alex) [inline docs/security.rst]: Good point.

On Fri, Feb 19, 2016 at 7:09 AM, Donald Stufft <notifications@github.com>
wrote:

> In docs/security.rst
> <https://github.com/pyca/cryptography/pull/2725#discussion_r53453458>:
>
> > +specific exploit, merely a reasonable threat model under which ``cryptography``
> > +could be attacked.
> > +
> > +To give a few examples of things we would consider security issues:
> > +
> > +* If a recipe, such as Fernet, made it easy for a user to bypass
> > +  confidentiality or integrity with the public API (e.g. if the API let a user
> > +  reuse nonces).
> > +* If, under any circumstances, we used a CSPRNG which wasn't fork-safe.
> > +* If ``cryptography`` used an API in an underlying C library and failed to
> > +  handle error conditions safely.
> > +
> > +Examples of things we wouldn't consider security issues:
> > +
> > +* Offering ECB mode for symmetric encryption. Though ECB is critically weak, it
> > +  is documented as being weak in our documentation.
>
> Should this explicitly mention offering ECB *in the hazmat layer*? I
> believe we would treat it as a security issue if the recipes layer allowed
> you to use ECB somehow yea?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2725/files#r53453458>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084

**Comment by dstufft** [comment]
:+1: This looks good to me.

**Comment by Ayrx** [comment]
:+1:

**Comment by alex** [comment]
Tests failing because hypothesis dropped 2.6 support. Working on a PR now.

**Comment by alex** [comment]
Jenkins, retest this please.

**Comment by reaperhulk** [comment]
:+1:

**Pull request merged**